cmd/fabric-ca-server/config.go

```diff
@@ -380,6 +382,11 @@ idemix:
   #  The value is expressed in the time.Duration format (see https://golang.org/pkg/time/#ParseDuration)
   noncesweepinterval: 15m
 
+  # Specifies the Elliptic Curve used by Identity Mixer.
+  # It can be any of: {"amcl.Fp256bn", "gurvy.Bn254", "amcl.Fp256Miraclbn"}.
+  # If unspecified, it defaults to 'amcl.Fp256bn'.
+  curveID: <<<IDEMIX_CURVE>>>
```

Comment:
I was expecting to see an associated CLI flag, similar to these:
https://github.com/hyperledger/fabric-ca/blame/main/docs/source/servercli.rst#L48-L50

Reply:
fixed